src/components/site/SummaryList.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+const SummaryList = props => {
```

Comment:
So I think the first step is to refactor this a bit, 

**Step 1: Add a `SummaryListItem`.**

This would contain the following props
- icon
- heading (the bold text)
- children (anything below the text)

**Step 2 - Remove any of the grid classes** 

We don't really need the grid here, it just add a bunch of markup. Try removing it and controlling the layout with [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Your icon and the "heading" will be aligned at the top.

**Step 3 - Change the static values below into data.**

For this component we should probably pass data as an `items` prop. This allows us to adjust quickly and potentially generate the html for this quickly down the road.

This would look something like this.

```jsx
const publicWorksSummaryItems = [
  {
    icon: "fas fa-map-marker-alt",
    heading: "Main Office",
    content: "<p>111 West Chesapeake Avenue<br /> Suite 307<br /> Towson, Maryland 21204</p>"
  }
];
<SummaryList item={publicWorksSummaryItems} />
```

Note: for content you will have to checkout [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml)

Also please see the comments below.

Reply:
done